draft-ietf-quic-multipath.md

```diff
@@ -481,6 +481,9 @@ it SHOULD NOT use the associated Path ID in future frames, except
 in ACK_MP frames for inflight packets and
 in MP_RETIRE_CONNECTION_ID frames for connection ID retirement.
 
+After a path is abandoned, the Path ID MUST NOT be reused
+for new paths, as the Path ID is part of the nonce calculation {{multipath-aead}}.
+
```

Comment:
We should also copy here the SHOULD requirement in the example text:
```
Endpoint SHOULD send MAX_PATHS frames {{max-paths-frame}} to raise
the limit of Path ID when endpoint finds there are not enough unused
Path ID (e.g. more than half of the available Path ID are used).
```

Reply:
PR #354 basically moves this text in the MAX_PATHS frame section. It might fit better here but my main intention is right now to have it only once normatively defined and not in a section where we shouldn't have normative language. I think I will make another paths later where we can again move things around. However, right now I'm really focusing on ensuring we have it defined only once and clearly.